catalog/task/collect-data/0.1/tests/test-collect-data-with-extra-data.yaml

```diff
@@ -107,26 +107,31 @@ spec:
           workspace: tests-workspace
     - name: check-result
       params:
-        - name: extraData
```

Comment:
I'd remove all this params stuff and just hardcode `cat $(workspaces.data.path)/extra_data.json` in the test

Reply:
fixed